content/release-10-18-0/streaming-analytics-10-18-0-bundle/10_18_0.md

```diff
@@ -0,0 +1,16 @@
+---
+weight: 40
+title: Release 10.18.0
+layout: redirect
+---
+
+### Apama correlator version
+
+This release of Cumulocity IoT Streaming Analytics includes the Apama version 10.15.3 correlator.
+EPL Apps developers should also refer to [What's New In Apama 10.15.3](https://documentation.softwareag.com/pam/10.15.3/en/webhelp/pam-webhelp/index.html#page/pam-webhelp%2Fco-WhaNewInApa_10153_top.html)
+in the Apama documentation.
+
+### Improvements in Analytics Builder
+
+The **Enabled** output port of the **Latch Values** block now generates a pulse signal when the block is enabled but was previously disabled.
```

Comment:
I think for release notes we need to say exactly what was not working and is now fixed rather than the expected behaviour?

The problem was : The Latch block did not start disabled when the Enable input was connected, which meant it produced a pulse on 'Disabled' output on the first false 'Value' input whereby it shouldn't. This has now been fixed.

please feel free to reword it.

Reply:
Oh, this looks like this is a fix. So it needs to go under a "Fixes" heading, with the usual table that we have for fixes.

Reply:
I've added your info, but also kept the previous one. Is this ok or do you want further changes?

Reply:
I am ok if you want to keep the previous text that you had added but i would suggest to remove the 'now'(2 places) from it because it use to do that previously (before fix) too but just not during startup of the block.

Reply:
The 2 "now"s have been removed. All good now?